target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -12,60 +12,39 @@ lantiq_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
-	alphanetworks,asl56026)
-		ucidef_add_switch "switch0" \
-			"2:lan" "3:lan" "6t@eth0"
-		;;
-	arcadyan,arv7519rw22)
-		ucidef_add_switch "switch0" \
-			"0:lan:5" "2:lan:3" "3:lan:4" "4:lan:1" "5:lan:2" "6t@eth0"
-		;;
+	alphanetworks,asl56026|\
 	arcadyan,vg3503j)
-		ucidef_add_switch "switch0" \
-			"2:lan:2" "4:lan:1" "6t@eth0"
+		ucidef_set_interfaces_lan "lan1 lan2"
```

Comment:
The function ```ucidef_set_interfaces_lan```does not exists, please use ```ucidef_set_interface_lan``` with interface without an s. This problem is in multiple places. ```ucidef_set_interfaces_lan_wan``` uses the plural interfaces.